snapvibe-infrastructure/src/main/kotlin/com/snap/vibe/global/security/SecurityConfig.kt

```diff
@@ -45,6 +44,9 @@ class SecurityConfig(
             .mvcMatchers(HttpMethod.DELETE, "/api/v1/post/{id}").authenticated()
             .mvcMatchers(HttpMethod.PATCH, "/api/v1/post/{id}").authenticated()
 
+            // comment
+            .mvcMatchers(HttpMethod.POST, "/api/v1/comment").authenticated()
```

Comment:
저도 잘 모르긴 하지만 저기에 api를 명시할때 이런식으로 해야하는거 아닌가요?
```suggestion
            .mvcMatchers(HttpMethod.POST, "/api/v1/comment/{post_id}").authenticated()
```

Reply:
수정했습니다 ~